### PR TITLE
Socint 186

### PR DESCRIPTION
**Pull request by KieranWardle**
Updated SurveyUpdate event to include new fields from schema v0.4.0

https://collaborate2.ons.gov.uk/jira/browse/SOCINT-186

Tested locally and in dev using rhsvc and rhcuc

### CODE REVIEW

File: event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SampleDefinitionColumn.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import java.util.List;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SampleDefinitionColumn {
```
Comment (rpcatling) [inline event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SampleDefinitionColumn.java:12]: This class not used, so can be deleted.

File: event-publisher/src/test/resources/uk/gov/ons/ctp/common/event/PackageFixture.FulfilmentEvent.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "header": {
-      "version": "v0.3_RELEASE",
```
Comment (rpcatling) [inline event-publisher/src/test/resources/uk/gov/ons/ctp/common/event/PackageFixture.FulfilmentEvent.json:4]: You missed some 0.4 changes needed in other elements below.

Also look at `FirestoreEventPersistenceTest.FulfilmentEvent.json` where another is also missed

File: event-publisher/src/test/resources/uk/gov/ons/ctp/common/event/PackageFixture.SurveyUpdateEvent.json [hunk 3]
```diff
@@ -12,7 +12,90 @@
   "payload": {
     "surveyUpdate": {
       "surveyId": "3883af91-0052-4497-9805-3238544fcf8a",
-      "name": "Excepteur veniam"
+      "name": "Excepteur veniam",
```
Comment (rpcatling) [inline event-publisher/src/test/resources/uk/gov/ons/ctp/common/event/PackageFixture.SurveyUpdateEvent.json:15]: did you want to change this to "LMS" for consistency ?

**Review by rpcatling — CHANGES_REQUESTED** [review]
minor fixes requested

File: event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SampleDefinitionColumn.java (hunk 1 above)
Comment (KieranWardle) [reply event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SampleDefinitionColumn.java]: Done

**Review by rpcatling — APPROVED** [review]

**Pull request merged**